app/controllers/user.js

```diff
@@ -0,0 +1,19 @@
+const { checkEmailExist, signUp } = require('../services/user');
+const logger = require('../logger');
+const errors = require('../errors');
+
+exports.signUp = (req, res, next) => {
```

Comment:
the controller should be as clean as possible, all logic should be in the services, I expect something like: 
```
serviceName(req|req.body|req.params...)
  .then(res => res.status(201).send(res))
  .catch(next);
```